src/plugins/ini/ini.c

```diff
@@ -21,8 +21,12 @@
 #include <kdbhelper.h>
 #include <kdbos.h>
 #include <inih.h>
+#include <ctype.h>
 #include "ini.h"
 
+
+char *keyNameGetOneLevel(const char *, size_t *);
```

Comment:
The API actually is obsolete because it needs reparsing which is already done. `keyUnescapedName` should be used instead. (But do not change it now, lets keep it that way for the release)